ci-operator/config/openshift/openshift-azure/private-preview-1.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: plugin-base
+    namespace: azure
+    tag: latest
+binary_build_commands: make all
+canonical_go_repository: github.com/openshift/openshift-azure
+images:
+- dockerfile_path: Dockerfile.sync
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/openshift-azure/sync
+  to: sync-pp
```

Comment:
@mjudeikis ptal, I think this will give us a separate tag for the private-preview branch which we can publish in quay.io

Reply:
should be good I think.